packages/contracts/contracts/voting/majority/MajorityVotingBase.sol

```diff
@@ -340,43 +338,48 @@ abstract contract MajorityVotingBase is
         return (_value * PCT_BASE) / _total;
     }
 
-    /// @notice Validates and sets the proposal vote settings.
-    /// @param _supportThreshold The support threshold in percent.
-    /// @param _minParticipation The minimum participation in percent.
-    /// @param _minDuration The minimum duration of a vote
-    /// @param _minProposerVotingPower The minimum voting power needed to create a proposal.
-    function _validateAndSetSettings(
-        uint64 _supportThreshold,
-        uint64 _minParticipation,
-        uint64 _minDuration,
-        uint256 _minProposerVotingPower
-    ) internal virtual {
-        if (_supportThreshold > PCT_BASE) {
-            revert PercentageExceeds100({limit: PCT_BASE, actual: _supportThreshold});
+    /// @notice Validates and updates the proposal majority voting settings.
+    /// @param _majorityVotingSettings The majority voting settings to be validated and updated.
+    function _validateAndUpdateSettings(MajorityVotingSettings calldata _majorityVotingSettings)
```

Comment:
Isn't the "validation" assumed? If the function updates the settings, I would call it _updateSettings (also, see the `setXXX` comment)

Reply:
agreed.

Reply:
Amended in 4e3d6689b50d901e133a26c23005b01badede8b2